app/browser/api/ledger.js

```diff
@@ -1852,7 +1881,63 @@ const onCallback = (state, result, delayTime) => {
   return state
 }
 
+const onReferralCodeRead = (err, code) => {
+  if (err) {
+    if (clientOptions.verboseP) {
+      console.error('read error: ' + err.toString())
+    }
+    return
+  }
+
+  if (!code) {
+    return
+  }
+
+  code = code.trim()
+
+  if (code.length > 2) {
```

Comment:
i thought the code was always going to be 6 characters? or can it be less?

Reply:
done